EIPS/eip-1238.md

```diff
@@ -0,0 +1,152 @@
+---
+eip: 1238
```

Comment:
```suggestion
eip: 4966
```

Reply:
Why suggest 4966? I wanted this PR to provide a solution for the discussion and post in #1238. Is that possible?

Reply:
I would be amenable to using that number if you can get approval from the original author of that issue.

Reply:
@nicola, I've been busy with defining a standard for your issue at #1238 over at #4973. And so far I've kept you as the co-author. Would you be willing to accept the documents attached to #4973 as the official ERC1238 standard? If not, we'll move forward with ERC4973. Please also let me know if you're interested in being a co-author. If not I'll remove you of course. Thank you.